.github/workflows/pull-request.yaml

```diff
@@ -180,3 +180,40 @@ jobs:
       - run: node_modules/.bin/swagger-cli validate crates/orderbook/openapi.yml
       - run: node_modules/.bin/swagger-cli validate crates/driver/openapi.yml
       - run: node_modules/.bin/swagger-cli validate crates/solvers/openapi.yml
+
+  run-flaky-test:
+    # to debug a flaky test set `if` to true and configure the flaky test in the `run` step
+    if: ${{ false }}
```

Comment:
How about using manual configs? Something like:
```
  workflow_dispatch:
    inputs:
      test_name:
        description: 'Test name to run'
        required: true
        default: 'local_node_no_liquidity_limit_order'
      test_threads:
        description: 'Number of test threads'
        required: false
        default: '1'
      retries:
        description: 'Number of retries'
        required: false
        default: '3'
```

Reply:
Also, `timeout-minutes` param.

Reply:
Ultimately this action should only ever be used in tandem with working on a flaky test.
So since you already have to work on the PR you might as well change the parameters in the github action while working on it.
Also I don't want to over-engineer it. We don't know how often we'll actually end up using this so spending more time now to make this "perfect" is not time well spent IMO.